Dockerfile.ubuntu

```diff
@@ -15,60 +15,61 @@ FROM ubuntu:20.04
 
 LABEL org.opencontainers.image.source https://github.com/helmfile/helmfile
 
-RUN apt-get update && \
-    apt-get install --no-install-recommends -y \
+RUN apt update -qq && \
+    apt install --no-install-recommends -y \
       ca-certificates \
       git bash curl jq wget openssh-client && \
     rm -rf /var/lib/apt/lists/*
 
-ARG HELM_VERSION="v3.10.2"
-ARG HELM_SHA256="2315941a13291c277dac9f65e75ead56386440d3907e0540bf157ae70f188347"
+ENV HELM_VERSION="v3.10.2"
 ARG HELM_LOCATION="https://get.helm.sh"
 ARG HELM_FILENAME="helm-${HELM_VERSION}-linux-amd64.tar.gz"
-
 RUN set -x && \
-    curl --retry 5 --retry-connrefused -LO ${HELM_LOCATION}/${HELM_FILENAME} && \
+    curl --retry 5 --retry-connrefused -LO "${HELM_LOCATION}/${HELM_FILENAME}{,.sha256sum}" && \
     echo Verifying ${HELM_FILENAME}... && \
-    sha256sum ${HELM_FILENAME} | grep -q "${HELM_SHA256}" && \
```

Comment:
I'd like to keep the usage and the embedding of HELM_SHA256 as is, because it's crucial for us to ensure that the helm binary downloaded in a docker build is exactly same as that is expected when the dockerfile is updated with a new helm release.

Reply:
@mumoshu make sense